static/app/views/performance/transactionSummary/transactionEvents/eventsTable.tsx

```diff
@@ -0,0 +1,341 @@
+        return undefined;
+      }
+
+      const nextEventView = eventView.sortOnField(field, tableMeta);
+      const queryStringObject = nextEventView.generateQueryStringObject();
+
+      return {
+        ...location,
+        query: {...location.query, sort: queryStringObject.sort},
+      };
+    }
+    const currentSort = eventView.sortForField(field, tableMeta);
+    // Event id and Trace id are technically sortable but we don't want to sort them here since sorting by a uuid value doesn't make sense
+    const canSort =
+      field.field !== 'id' &&
+      field.field !== 'trace' &&
+      field.field !== SPAN_OP_RELATIVE_BREAKDOWN_FIELD &&
+      isFieldSortable(field, tableMeta);
+
+    const currentSortKind = currentSort ? currentSort.kind : undefined;
+    const currentSortField = currentSort ? currentSort.field : undefined;
+
+    if (field.field === SPAN_OP_RELATIVE_BREAKDOWN_FIELD) {
+      title = (
+        <OperationSort
+          title={OperationTitle}
+          eventView={eventView}
+          tableMeta={tableMeta}
+          location={location}
+        />
+      );
+    }
+
+    const sortLink = (
+      <SortLink
+        align={align}
+        title={title || field.field}
+        direction={currentSortKind}
+        canSort={canSort}
+        generateSortLink={generateSortLink}
+        onClick={() => this.onSortClick(currentSortKind, currentSortField)}
+      />
+    );
+    return sortLink;
+  }
+
+  renderHeadCellWithMeta = (tableMeta: TableData['meta']) => {
+    const columnTitles = this.props.columnTitles ?? COLUMN_TITLES;
+    return (column: TableColumn<keyof TableDataRow>, index: number): React.ReactNode =>
+      this.renderHeadCell(tableMeta, column, columnTitles[index]);
+  };
+
+  handleResizeColumn = (columnIndex: number, nextColumn: GridColumn) => {
+    const widths: number[] = [...this.state.widths];
+    widths[columnIndex] = nextColumn.width
+      ? Number(nextColumn.width)
+      : COL_WIDTH_UNDEFINED;
+    this.setState({widths});
+  };
+
+  render() {
+    const {eventView, organization, location, setError} = this.props;
+
+    const {widths} = this.state;
+    const containsSpanOpsBreakdown = eventView
+      .getColumns()
+      .find(
+        (col: TableColumn<React.ReactText>) =>
+          col.name === SPAN_OP_RELATIVE_BREAKDOWN_FIELD
+      );
+    const columnOrder = eventView
+      .getColumns()
+      .filter(
+        (col: TableColumn<React.ReactText>) =>
+          !containsSpanOpsBreakdown || !col.name.startsWith('spans')
+      )
+      .map((col: TableColumn<React.ReactText>, i: number) => {
+        if (typeof widths[i] === 'number') {
+          return {...col, width: widths[i]};
+        }
+        return col;
+      });
+
+    return (
+      <div>
+        <DiscoverQuery
+          eventView={eventView}
+          orgSlug={organization.slug}
+          location={location}
+          setError={setError}
+          referrer="api.performance.landing-table"
```

Comment:
This would need to be renamed as it's already used elsewhere.

Reply:
Good catch 😄